code/modules/supermatter/supermatter.dm

```diff
@@ -464,4 +468,97 @@
 /obj/machinery/power/supermatter/clone(var/obj/machinery/power/supermatter/O)
 	id_tag = O.id_tag
 	set_frequency(O.frequency)
-	return 1
\ No newline at end of file
+	return 1
+
+/obj/machinery/computer/supermatter
+	name = "Supermatter Monitoring Computer"
```

Comment:
silly capitalization

Reply:
For mysterious reasons, it is standard for computers to have capitalized names (Communications, Security Cams, Atmos Alert, ID computer, Hologram comp, AI Fixer, AI Upload, Med Records, etc etc)